docker-compose.yml

```diff
@@ -69,6 +89,13 @@ services:
       - packages:/var/opt/warehouse/packages
     ports:
       - "80:8000"
+    links:
```

Comment:
is this just making the links explicit, I'm not familiar with why this would be necessary for web but not worker in that case.

Reply:
Yep, correct. My understanding (and experience) is that all containers are exposed by name on the same network by default; `links` just forces `docker-compose` to start `vault` before it starts `web`.

`worker` should also probably have a `links` entry for it, since we'll be accessing vault via Celery tasks. So I'll add it 🙂

Reply:
It's advised to not use `links`: https://docs.docker.com/compose/compose-file/#links

Reply:
Thanks for the ref! The other Docker page that documents `links` doesn't mention their deprecation: https://docs.docker.com/compose/networking/#links, so I missed it. I'll drop these.